src/core/on-chain/index.ts

```diff
@@ -68,7 +71,7 @@ export const OnChainMixin = (superclass) =>
     }
 
     async updatePending(lock): Promise<void> {
```

Comment:
I'm wondering if we now ever needs this function any more if we're passing awaiting on the parent?

Reply:
the order is `OnChainMixin(LightningMixin(UserWallet))` so we need to call the super